cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -156,12 +164,22 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
       -- tip
       do
         -- to stdout
-        _ :: QueryTipLocalStateOutput <- H.noteShowM $ execCliStdoutToJson execConfig [ eraName, "query", "tip" ]
-        -- to a file
+        tips :: QueryTipLocalStateOutput <- H.noteShowM $ execCliStdoutToJson execConfig [ eraName, "query", "tip" ]
+        let changes = Map.fromList (map (, "<redacted>") ["hash", "block", "epoch", "slot", "slotInEpoch", "slotsToEpochEnd"])
```

Comment:
Use `import GHC.Exts (IsList(..))`
```suggestion
        let changes = fromList (map (, "<redacted>") ["hash", "block", "epoch", "slot", "slotInEpoch", "slotsToEpochEnd"])
```

Reply:
@carbolymer> Used the `import` and dropped the `Map` qualification :+1: Thanks for letting me know about `IsList`.